doc/man3/SSL_CONF_cmd.pod

```diff
@@ -85,6 +85,18 @@ its preference list. This usually indicates a client without AES hardware
 acceleration (e.g. mobile) is in use. Equivalent to B<SSL_OP_PRIORITIZE_CHACHA>.
 Only used by servers. Requires B<-serverpref>.
 
+=item B<-ignore_unexpected_eof>
+
+Some TLS implementations do not send a close_notify alert on shutdown. Before
+OpenSSL 3.0, B<SSL_ERROR_SYSCALL> with errno=0 was reported in such cases. This
+behaviour contradicts the B<SSL_ERROR_SYSCALL> documentation. Since OpenSSL 3.0
+the returned error is B<SSL_ERROR_SSL> with a meaningful error on the error
+stack. When this option is enabled an B<SSL_ERROR_ZERO_RETURN> error is
+returned in case an EOF is detected and the close_notify alert was not
+received. You should only enable this option if the protocol running over TLS
```

Comment:
Here is a replacement suggestion:
Some TLS implementations do not send the mandatory close_notify alert on shutdown. If the application tries to wait for the close_notify alert but the peer closes the connection without sending it, an error is generated.  When this option is enabled the peer does not need to send the close_notify alert and a closed connection will be treated as if the close_notify alert was received. For more information on shutting down a connection, see L<SSL_shutdown>.

You should only [...]

SSL_shutdown() should probably mention the option.

Reply:
Done.